.gitignore

```diff
@@ -76,3 +76,6 @@ typings/
 .env.development.local
 .env.test.local
 .env.production.local
+
+#opt/datawork
+/opt
```

Comment:
/opt shouldn't not be accessible from the git folder so are you sure this is needed ? Also be sure to always have a newline at the end of each file :wink: this is way here in github diff you have a red symbol